packages/Students/StudentSchedule/src/Services/ScheduleService.php

```
@@ -2,108 +2,74 @@
-
-    /**
-     * Gộp sự kiện từ 4 nguồn trong khoảng [$from, $to].
-     * Mỗi sự kiện: (object){ type, title, at(Carbon), end(Carbon|null), url, classroom, tone, icon }
-     */
-    public function eventsBetween(int|string $studentId, Carbon $from, Carbon $to): Collection
-    {
-        $student = User::query()->findOrFail($studentId);
-
-        return $this->calendar->events(new CalendarQuery(
+        [$from, $to] = $this->range($anchor, $view);
+        $kinds = collect($filters['kinds'] ?? [])->map(fn (string $kind) => CalendarEventKind::from($kind))->all();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Mindigo\AcademicCalendar\Enums\CalendarEventKind' in method 'workspace'. (undefined)

(StaticAccess)